number_theory/modular.h

```diff
@@ -46,6 +63,31 @@ struct ModulusWrapper {
 
 }  // namespace modular_internal
 
+// Returns the modular inverse of |number| in modulo |mod| if exists.
+// Otherwise, throws invalid_argument exception.
```

Comment:
1. `|mod|` -> `|modulus|`
2. It throws domain_error.

Reply:
I think that's one of the hard parts about exceptions that not only we need to mention when it throws which kind of exception, but also we should be careful about indirect exceptions from called functions if we are not handling or rethrowing them.

Maybe we can try to adapt to Google's Python style guide since they use exception:
https://google.github.io/styleguide/pyguide.html#doc-function-raises